<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurrentlyPlayingWidget</class>
 <widget class="QWidget" name="CurrentlyPlayingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="currentlyPlayingGroupBox">
     <property name="title">
      <string>Current Song</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="artistLabel">
        <property name="text">
         <string>Artist: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="songProgressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="playPausePushButton">
          <property name="text">
           <string>Play/Pause</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="prevPushButton">
          <property name="text">
           <string>Prev</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextPushButton">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="currentPlayListGroupBox">
     <property name="title">
      <string>Current Playlist</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeView" name="currentPlayListTreeView">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <property name="allColumnsShowFocus">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="selectSongPushButton">
          <property name="text">
           <string>Play selected</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="selectSongPrevPushButton">
          <property name="text">
           <string>Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectSongNextPushButton">
          <property name="text">
           <string>Down</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectSongPageUpPushButton">
          <property name="text">
           <string> Page Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectSongPageDownPushButton">
          <property name="text">
           <string>Page Down</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectSongFirstPushButton">
          <property name="text">
           <string>Top</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectSongLastPushButton">
          <property name="text">
           <string>Bottom</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
